best-practices/README.md

```diff
@@ -13,6 +13,12 @@ General
 * [Keep the code simple](http://www.readability.com/~/ko2aqda2).
 * Avoid premature optimizations. Get a version 1.0 out there as soon as you
   can. Until you have some users to measure, you're optimizing based on guesses.
+* Use `bin/setup` for initial setup of the project and for development
+  environment seed data.
+* Use `bin/server` for starting all the processes required to run the project
+  in a development environment, when suitable.
+* Use `bin/deploy` for deploying the project to staging and production
```

Comment:
I haven't actually run a deploy command for a project in a while, having everything automated by semaphore. Has anyone? /cc @subvisual/developers 

I propose that we simply get rid of this and related to deploys just say that it should be set up on the CI (specifics on that may go on another guide) and from there all deploys should go by the CI, or if applicable, review apps.

Reply:
Following up on this forgotten conversation, for some of our projects Heroku is not an option and deployments from Semaphore might also not be (we have at least one such project at the moment).

Given that we use SemaphoreCI and they recently released the first version of their CLI tool (which should include deployment control in the future), and most of us already use parity with Heroku projects, I vote for including `bin/deploy` in all projects for the sake of having a common way of triggering the deployments. Be them through SemaphoreCI, Git pushes or whatever, you know that if you just run `bin/deploy` it will trigger the deployment (or maybe guide you through it?).

Reply:
@zamith @naps62 ping on this.